src/app/api/schemas.py

```diff
@@ -143,11 +148,11 @@ class MediaOut(MediaIn, _CreatedAt):
 class InstallationIn(BaseModel):
     device_id: int = Field(..., gt=0)
     site_id: int = Field(..., gt=0)
-    elevation: float = Field(..., gt=0., lt=10000)
-    lat: float = Field(..., gt=-90, lt=90)
-    lon: float = Field(..., gt=-180, lt=180)
-    yaw: float = Field(..., gt=-180, lt=180)
-    pitch: float = Field(..., gt=-90, lt=90)
+    elevation: float = Field(..., gt=0., lt=10000, example=1110)
+    lat: float = Field(..., gt=-90, lt=90, example=48.8534)
+    lon: float = Field(..., gt=-180, lt=180, example=2.3488)
+    yaw: float = Field(..., gt=-180, lt=180, example=42)
+    pitch: float = Field(..., gt=-90, lt=90, example=16)
     start_ts: datetime = None
     end_ts: datetime = None
```

Comment:
over there

Reply:
same as for ids, because dates are filled with now() in docs, I have not add some at first. But I can add some examples dates, will make it clearer !

Reply:
Oh you're right, ignore my previous comment then!